packages/lab/src/components/Flow/Node/index.ts

```diff
@@ -1 +1,3 @@
 export * from "./Node";
+export * from "./ParamsNode";
+export * from "./DaNode";
```

Comment:
The refactor into multiple components is nice, but I'm not sure about the naming...

We shouldn't an exported API so specific to a particular use-case that it even has its name. If it really is only suitable for that case, it should be a sample. If it's more generic than that, let's give it a more generic name.

We should have a `HvFlowBaseNode` which assumes very little (like the new `HvFlowNode`), similar to our other `Base*` components

Reply:
I think the `DaNode` should only exist in our samples and not as an exported component by the UI Kit since it's specific to a particular use-case.
Also, I agree that we should have a `HvFlowBaseNode` by renaming the new `HvFlowNode`.